src/pool/srv_metrics.c

```diff
@@ -42,21 +42,44 @@ ds_pool_metrics_alloc(const char *path, int tgt_id)
 	if (metrics == NULL)
 		return NULL;
 
-	rc = d_tm_add_metric(&metrics->open_hdl_gauge, D_TM_GAUGE,
-			     "Number of open pool handles", "hdls",
-			     "%s/pool_handles", path);
-	if (rc != 0)
-		D_ERROR("Couldn't add open handle gauge: "DF_RC"\n", DP_RC(rc));
-
 	rc = d_tm_add_metric(&started, D_TM_TIMESTAMP,
 			     "Last time the pool started", NULL,
 			     "%s/started_at", path);
 	if (rc != 0) /* Probably a bad sign, but not fatal */
-		D_ERROR("Failed to add started_timestamp metric, " DF_RC "\n",
-			 DP_RC(rc));
+		D_WARN("Failed to create started_timestamp metric: "DF_RC"\n", DP_RC(rc));
 	else
 		d_tm_record_timestamp(started);
 
+	rc = d_tm_add_metric(&metrics->evict_total, D_TM_COUNTER,
+			     "Total number of pool handle evict operations", "ops",
```

Comment:
Question - Similar to the container operations, since these are only incremented on success, do we want to include "successful" in the descriptions?

Reply:
all metrics are for successful operations actually (unless stated otherwise). Not sure whether we should specify it every time ...